pkg/virt-operator/application.go

```diff
@@ -163,6 +164,9 @@ func Execute() {
 	if err != nil {
 		golog.Fatalf("Error determining cluster type: %v", err)
 	}
+	if err := os.Setenv(util.OnOpenshiftEnvName, strconv.FormatBool(onOpenShift)); err != nil {
```

Comment:
I am not sure if this is the best way to propagate info that Kubevirt is deployed on OpenShift, with this env I don't have to change config logic too much, but I have mixed feelings about it. Any opinion here?

Reply:
mh, I indeed don't like it this way. Can't you just cache the isOnOpenShift value in pkg/virt-operator/util/client.go, where it's discovered?
Other than that David was faster than me ;)

Reply:
Yeah, I will change it, this solution sucks.

Reply:
i think we should cache the result somewhere outside of the install strategy.

Caching it in `pkg/virt-operator/util/client.go` would be fine, but that would require synchronization using locking. 

Another option is to add a boolean to the `pkg/virt-operator/util/types.go` files Stores struct called soemthing like "IsOnOpenshift". Then in `pkg/virt-operator/application.go` you could do something like this in the onOpenShift if/else that already exists. 

```
        if onOpenShift {                                                                       
                log.Log.Info("we are on openshift")                           
                app.informers.SCC = app.informerFactory.OperatorSCC()                          
                app.stores.SCCCache = app.informerFactory.OperatorSCC().GetStore()
                app.stores.IsOnOpenshift = true
        } else {
                log.Log.Info("we are on kubernetes")
                app.informers.SCC = app.informerFactory.DummyOperatorSCC()    
                app.stores.SCCCache = app.informerFactory.DummyOperatorSCC().GetStore()
        } 
```

Reply:
done